test/unit/bodyCullTest.js

```diff
@@ -8,6 +8,10 @@ import { clone } from '../utils'
 import moment from 'moment'
 import should from 'should'
 
+import { connectionDefault } from '../../src/config'
+const MongoClient = connectionDefault.client
+let db = null
```

Comment:
@BMartinos Could this variable could be declared further within the `describe` closer to where it's used? That would reduce it's scope and make it easier to follow.

Reply:
Done in 707d03e